Assets/Scripts/Objects/InsertAndRemoveTape.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Video;
+
+public class InsertAndRemoveTape : MonoBehaviour
```

Comment:
Might be good to rename this to something like "VHS Player" or "Tape Manager", since we need something like that anyway to control what versions of the tapes are being played.

Reply:
Ok I'll rename